packages/angular_devkit/build_ng_packagr/src/build/index_spec_large.ts

```diff
@@ -18,10 +18,16 @@ import {
 } from '@angular-devkit/core'; // tslint:disable-line:no-implicit-dependencies
 import { map, take, tap } from 'rxjs/operators';
 
+export const ivyEnabled = process.argv.includes('--ivy');
```

Comment:
I don't think this ever runs. We use a glob for the ivy test large:
https://github.com/angular/angular-cli/blob/607e6f7cfd7b5f16231eae968fbf36e920c169e0/.circleci/config.yml#L340-L345

Can you alter the glob to include build_ng_packagr too?

Reply:
Good catch!